### DISCUSSION TITLE
PrimeReact page alignment CSS issue

### DISCUSSION THREAD

**Discussion by AbhilashKoluguri** [primefaces/community#145] [PrimeReact]
Hello Everyone,

We are facing the below issue from a day, all of our React Application built with prime react has alignment and styling issues, it behaves like if CSS is not loaded properly, this happened without any code changes,
Advice if any solution is available,
![image](https://github.com/primefaces/community/assets/43523526/3f13cb58-7868-4252-afb9-5be7cdd53b49)

We use these external style classes,
<link rel="stylesheet" href="https://unpkg.com/primeicons/primeicons.css" />
  <link rel="stylesheet" href="https://unpkg.com/primereact/resources/themes/saga-blue/theme.css" />
  <link rel="stylesheet" href="https://unpkg.com/primereact/resources/primereact.min.css" />
 <script src="https://unpkg.com/primereact/core/core.min.js"></script>
  <script src="https://unpkg.com/primereact/column/column.min.js"></script>
  <script src="https://unpkg.com/primereact/datatable/datatable.min.js"></script>
  <script src="https://unpkg.com/primereact/chart/chart.min.js"></script>
  <script src="https://unpkg.com/primereact/breadcrumb/breadcrumb.min.js"></script>
  <script src="https://unpkg.com/primereact/toast/toast.min.js"></script>
  <script src="https://unpkg.com/primereact/scrolltop/scrolltop.min.js"></script>
  <script src="https://unpkg.com/primereact/panel/panel.min.js"></script>

**Comment by NekkalapuVineeth** [comment]
Hi Everyone 

Even am facing the same issue since one day getting the below error which i can see in console
<img width="931" alt="Issue" src="https://github.com/primefaces/community/assets/146416293/f2db3acc-d6f8-4c76-a0e1-b60a81e3f28a">

Our application is loading in this formate

![image](https://github.com/primefaces/community/assets/146416293/0d9c0316-46f9-468b-81bf-552e882605d3)

**Reply by habubey** [reply]
Hi there! Which version of PrimeReact are you using?

**Reply by NekkalapuVineeth** [reply]
Hi @habubey 
Please find the our primereact versions
 "primeicons": "^5.0.0",
 "primereact": "^9.6.2",

**Comment by mertsincan** [comment]
Sorry for the delayed response! Could you please try it with the latest PrimeReact version? For now, closed. If the problem persists, please reopen it with a stackblitz link for us to replicate.

Best Regards,
